standard/classes.md

```diff
@@ -2177,7 +2180,7 @@ Output parameters are typically used in methods that produce multiple return val
 >     static void Main()
 >     {
 >         string dir, name;
->         SplitPath("c:\\\Windows\\\\System\\\\hello.txt", out dir, out name);
+>         SplitPath("c:\\Windows\\System\\hello.txt", out dir, out name);
```

Comment:
Might be worth using a verbatim string literal here? `SplitPath(@"c:\Windows\System\hello.txt", out dir, out name);`